pulp_smash/tests/pulp3/pulpcore/api_v3/test_crd_artifacts.py

```diff
@@ -51,6 +54,23 @@ def test_01_create(self):
             hashlib.sha256(files['file']).hexdigest()
         )
 
+    def test_01_create_no_match_data(self):
```

Comment:
When I read this method name, I get the sense that no `data` is being passed in. This method does pass in data, though. I think method naming could be improved. Something like this might be better:

```
test_01_create_no_data
test_01_create_valid_data
test_01_create_invalid_data
```